kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/model/VirtualClusterModel.java

```diff
@@ -83,7 +83,7 @@ public VirtualClusterModel(String clusterName,
     }
 
     public void logVirtualClusterSummary() {
-        var upstreamHostPort = targetCluster.bootstrapServersList().get(0);
+        var upstreamHostPort = targetCluster.bootstrapServersList();
```

Comment:
We should probably do a [`String.join`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/String.html#join(java.lang.CharSequence,java.lang.CharSequence...)) here to format the list nicely.

Reply:
It looks like log4j already does a reasonable job
```
kroxylicious  | 2025-08-13 16:02:10 <main> INFO  io.kroxylicious.proxy.model.VirtualClusterModel - Virtual Cluster 'demo' - gateway summary
kroxylicious  | 2025-08-13 16:02:10 <main> INFO  io.kroxylicious.proxy.model.VirtualClusterModel - Gateway: mygateway, Downstream localhost:9192 => Upstream [kafka-1:9092, kafka-2:9092, kafka-3:9092]
```